pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -424,6 +427,8 @@ func (am *Alertmanager) applyConfig(cfg *apimodels.PostableUserConfig, rawConfig
 	for name := range integrationsMap {
 		stage := am.createReceiverStage(name, integrationsMap[name], am.waitFunc, am.notificationLog)
 		routingStage[name] = notify.MultiStage{meshStage, silencingStage, timeMuteStage, inhibitionStage, stage}
+		// TODO: true?
```

Comment:
I'm not sure I understand what this TODO is about, but don't forget it!

Reply:
It's basically "is it ok to always pass `true` as the value for the `active` field in the receiver we create?"
I could've been a little more descriptive lol